src/code/stores/presence-store.ts

```diff
@@ -10,7 +10,7 @@ export const PresenceStore = types
   })
   .views(self => ({
     get selected(): IPresence | undefined{
-      return self.presences.get(gFirebase.user.uid);
+      return self.presences.get(gFirebase.user && gFirebase.user.uid || "");
```

Comment:
Obviously the `|| ""` safeguard is appeasing our type checker.  

To me, it says "lookup the value associated with null-string", which seems wrong.   :)

Reply:
I added an extra set of parentheses in an attempt to visually separate the "looking up a value" part of the expression from the "defaulting in case of failure" part of the expression.